fe/fe-core/src/main/java/org/apache/doris/planner/external/HiveScanNode.java

```diff
@@ -192,12 +195,50 @@ private void getFileSplitByPartitions(HiveMetaStoreCache cache, List<HivePartiti
                 for (HiveMetaStoreCache.HiveFileStatus status : fileCacheValue.getFiles()) {
                     allFiles.addAll(splitFile(status.getPath(), status.getBlockSize(),
                             status.getBlockLocations(), status.getLength(), status.getModificationTime(),
-                            isSplittable, fileCacheValue.getPartitionValues()));
+                            isSplittable, fileCacheValue.getPartitionValues(), fileCacheValue.getAcidInfo()));
                 }
             }
         }
     }
 
+    private List<Split> splitFile(Path path, long blockSize, BlockLocation[] blockLocations, long length,
```

Comment:
Better to find a way to extract the common part of `splitFile` and the same method defined in `FileScanNode`, or it is hard to maintain.